apps/feedbacks/views.py

```diff
@@ -26,6 +25,24 @@ def index(request):
     }
     return render(request, 'feedbacks/index.html', context)
 
+
 def feedback_submitted(request):
     return render(request, 'feedbacks/feedback_submitted.html')
 
+
+@csrf_exempt
+def contact(request):
+    if request.method == 'POST':
+        try:
+            name = request.POST['full_name']
+            phone = request.POST['phone']
+            email = request.POST['email_id']
+            msg = request.POST['msg']
+            contact_data = Contact(name=name, phone=phone, email=email, message=msg)
+            contact_data.save()
+            messages.success(request, 'Your Contact information is saved successfully.')
```

Comment:
It should be, `We got your message. Someone from our team will shortly contact you.`